overlord/ifacestate/ifacemgr_test.go

```diff
@@ -336,6 +336,46 @@ func (s *interfaceManagerSuite) TestDoSetupSnapSecuirtyKeepsExistingConnectionSt
 	})
 }
 
+func (s *interfaceManagerSuite) TestDoSetupSnapSecuirtyReloadsConnectionsWhenInvokedOnPlugSide(c *C) {
```

Comment:
LGTM, but let's please rename these functions to be at most half that len. Let's use comments to document them instead.